### DISCUSSION TITLE
Show Existing tables

### DISCUSSION THREAD

**Discussion by diegoserranobst** [AntlerVC/firetable#1211] [Support / Q&A] [answered]
I have some collections already stored in my Firestore database, but I am not able to view them on the "Table" section. Is it supposed to be visible on this section? If yes, I am not able to see any tables currently.

**Comment by harinij** [comment, marked as answer]
@diegoserranobst For existing collections you still need to create a table to see them on the tables list. Click + and simply pick the collection from the drop-down list. Doc on creating a table is here: https://docs.rowy.io/how-to/create-table
